core/includes/config.inc

```diff
@@ -262,6 +261,7 @@ function config_install_default_config($module, $config_name = NULL) {
       $file = array_pop($parts);
       $file_config_name = str_replace('.json', '', $file);
       if (is_null($config_name) || $file_config_name === $config_name) {
+        $storage = new ConfigFileStorage($module_config_dir);
```

Comment:
Not sure what this does and how things work, so cannot comment on this change (other than making this comment here that is 😄)

Reply:
I don't know either yet, but the proposed change make sense to me.

Reply:
This seems to be a mishap. @BWPanda fixed it. Everything makes sense now 😄